### DISCUSSION TITLE
141

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#76] [General]
# 潮流周刊第141期 - 西湖中央

封面图来源于周末陪家人在西湖散散步的随手拍，我比较喜欢人少的时候去，找一个有风的椅子，可以安静坐很久，相比走来走去要舒服不少。 

https://weekly.tw93.fun/posts/141-%E8%A5%BF%E6%B9%96%E4%B8%AD%E5%A4%AE

**Comment by ricardoo9** [comment]
weekly相当于推特的汇总版，加油

**Reply by tw93** [reply]
哈哈哈，谢谢喜欢～

**Comment by Wuli-Blizzard** [comment]
感谢分享！
